sdk/spring/spring-messaging-azure-eventhubs/src/main/java/com/azure/spring/eventhubs/support/converter/EventHubsMessageConverter.java

```diff
@@ -74,14 +77,16 @@ protected EventData fromByte(byte[] payload) {
     @Override
     protected void setCustomHeaders(MessageHeaders headers, EventData azureMessage) {
         super.setCustomHeaders(headers, azureMessage);
+        Set<String> ignoredHeaders = new HashSet<>();
         headers.forEach((key, value) -> {
-            if (SYSTEM_HEADERS.contains(key)) {
-                LOGGER.warn("System property {}({}) is not allowed to be defined and will be ignored.",
-                    key, value);
+            if (IGNORED_HEADERS.contains(key)) {
+                ignoredHeaders.add(key);
             } else {
                 azureMessage.getProperties().put(key, value.toString());
             }
         });
+
+        ignoredHeaders.forEach(header -> LOGGER.info("Message header {} will be ignored.", header));
```

Comment:
LOGGER.info("Message headers {} will be ignored", ignoredHeaders) 
only one line log for this?